kolibri/content/serializers.py

```diff
@@ -14,11 +14,25 @@
 from kolibri.content.models import Language
```

Comment:
There are two main improvements in this serializer:
1. Sped up the individual calculations through a lot of trial, error, diagnosing, tweaking.
2. Made the different extra fields individually loadable -- we don't need to load 3 of the 4 on the channel list page, so we can save a lot of time there by just requesting the total storage used.